drivers/video/mxc_ipuv3_fb.c

```diff
@@ -425,8 +425,9 @@ static int mxcfb_map_video_memory(struct fb_info *fbi)
 	debug("allocated fb @ paddr=0x%08X, size=%d.\n",
 		(uint32_t) fbi->fix.smem_start, fbi->fix.smem_len);
 
-
+#if defined(CONFIG_LCD) || defined(CONFIG_VIDEO)
 	gd->fb_base = fbi->fix.smem_start;
+#endif
```

Comment:
Shouldn't this entire file (ipuv3_fb) not be built when CONFIG_VIDEO is not set?